recipes/treelite/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "treelite" %}
+{% set version = "0.92" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmlc/treelite/archive/{{ version }}.tar.gz
+  sha256: 17961a4f4c5933630dfee5c7129180e9d69660f3efa05f87bc8b168081bf3fd7
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

Comment:
This can be dropped. The oldest `python` version we support is 3.6. We already dropped 2.7 and 3.5 in conda-forge previously.

Reply:
That's good to know. Let me remove it.